src/Exceptionless.Core/Models/Data/RequestInfo.cs

```diff
@@ -114,6 +114,11 @@ public static class KnownDataKeys {
             public const string OSMajorVersion = "@os_major_version";
 
             public const string IsBot = "@is_bot";
+
+            /// <summary>
+            /// The IP address that submitted the event to our api.
+            /// </summary>
+            public const string RequestIpAddress = "@request_ip";
```

Comment:
I don't understand this. We already have ClientIpAddress. If this is just the IP address of the Exceptionless client that submitted the events to our API then: a) it should not be on RequestInfo and b) we already have that from the HTTP headers of the incoming event post and that gets put into the event payload data.

Reply:
yeah, this is passed from the client payload data. We put it in the client IP address because most of the time (very large percentage) there is no ip because there is no way to get the remote ip without doing network calls.

Reply:
So I have no idea why we would need to add it to the RequestInfo. RequestInfo is specifically for the HTTP request that is attached to the event. Not the Exceptionless client request that is submitting the events to our API.

Reply:
I think we were always trying to tell you what server / machine that the request happened on in case there wasn't an external address.